ponder.schema.ts

```diff
@@ -73,13 +89,15 @@ export default createSchema((p) => ({
     id: p.string(),
     chain: p.string(),
     hearter: p.string(),
-    memeTokenId: p.string().references("MemeToken.id"),
+    memeTokenId: p.string().references("MemeToken.id").optional(),
+    memeNonce: p.bigint().optional(),
```

Comment:
unable to reference here, unsure if this is foreign keys database, or is an API layer reference

Reply:
This should be memeToken no? In the old contract the Heart event had that as the second value of the event, now its memeNonce

Reply:
agree, this was renamed previously, but confusing, removing in follow up